inspirehep/dojson/jobs/fields/bd1xx.py

```diff
@@ -51,9 +51,9 @@ def date_closed(self, key, value):
             elif "www" in val.get('l') or "http" in val.get('l'):
```

Comment:
Can you commit this snippet first in utils?
```
def force_single_element(obj):
    """Force an object to a list and return the first element."""
    lst = force_list(obj)
    if lst:
        return lst[0]
    return None
```

Just to make sure that the field 1 is never repeated